docs/annotations.md

```diff
@@ -156,6 +157,57 @@ spec:
           servicePort: 80
 ```
 
+## AppGw SSL Certificate
+
+The SSL certificate [can be configured to Application Gateway](https://docs.microsoft.com/en-us/cli/azure/network/application-gateway/ssl-cert?view=azure-cli-latest#az-network-application-gateway-ssl-cert-create) either from a local PFX cerficate file or a reference to a Azure Key Vault unversioned secret Id.
+When the annotation is present with a certificate name and the certificate is pre-installed in Application Gateway, Kubernetes Ingress controller will create a routing rule with a HTTPS listener and apply the changes to your App Gateway.
+`appgw-ssl-certificate` annotation can also be used together with `ssl-redirect` annotation in case of SSL redirect.
+
+> **Note**
+* Annotation "appgw-ssl-certificate" will be ignored when TLS Spec is defined in ingress at the same time.
+* If a user wants different certs with different hosts(multi tls certificate termination), they would need to define different ingress resources.
+
+### Use Azure CLI to install certificate to Application Gateway
+* Configure from a local PFX certificate file
+```bash
+az network application-gateway ssl-cert create -g $resgp  --gateway-name $appgwName -n mysslcert --cert-file \path\to\cert\file --cert-password Abc123
+```
+
+* Configure from a reference to a Key Vault unversioned secret id
```

Comment:
provide a link to the other document here since keyvault requires more steps.

Reply:
> provide a link to the other document here since keyvault requires more steps.

provide a link to the feature page as well as configure via powershell